app/views/site/about.html.erb

```diff
@@ -64,7 +64,6 @@
   services to keep the site running, including:
 
   <ul>
-    <li><a href="https://heroku.com">Heroku</a>
```

Comment:
even if they no longer do, they did offer credits for a long time. do they deserve some highlight for previous donations?

Reply:
True. I don't have a strong opinion about it. I'd probably just as soon get rid of it, since we have been publicly thanking them up until this point.